content/post/2022-01-21-friday-hacks-214.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Friday Hacks #214, January 21"
+date: 2022-01-15 00:35:42.301617
+author: Simon Julian Lauw
+url: /2022/01/friday-hacks-214
+summary: GPU.js and Hack&Roll 2022 Projects
+nofh: true
+supportedByJetBrains: true
+---
+
+{{< friday_hack_header
+    venue="Seminar Room 8 (SR8), COM1 02-08 & Online on Zoom (Hybrid)"
+    date="January 21"
+    food="pizza"
+    rsvp_link="https://google.com" >}}
```

Comment:
i assume this should be the google form link?

Reply:
Yes, my bad on forgetting to change this link